src/projector.ts

```diff
@@ -29,9 +29,17 @@ export interface ProjectorOptions extends ParentMixinOptions<Child>, EventedOpti
 	 *
 	 * Only applies if `autoAttach` is `true`
 	 */
+	// FIXME Is this used? What about replace?
```

Comment:
or `boolean | 'append' | 'merge' | 'replace'`? Where `true` and `'merge'` are equivalent?